lesson10.js

```diff
@@ -0,0 +1,80 @@
+class Item{
+    constructor(name, price, count){
+        this.name = name;
+        this.price = price;
+        this.count = count;
+    }
+    render(){
+        return `<div class = "item"><h4>${this.name}</h4><span>${this.price}</span>
+        <button class = "deleteProd">-</button><span class = "count">${this.count}</span><button class = "addProd">+</button></div>`;
+    }
+}
+
+class Catalog {
+    items
+    constructor(url){
+        this.url = url;
+        this.items = [];
+    }
+    fetchGoods(){
+        return fetch(`http://localhost:3000${this.url}`).then((response)=>
```

Comment:
Будь внимательнее с оформлением. Здесь и во многих местах не хватает пробелов - это важно для читаемости кода
```suggestion
        return fetch(`http://localhost:3000${this.url}`).then((response) =>
```